development/setup/amplifysetup.md

```diff
@@ -1 +1,104 @@
 # Owner - Kamal
+
+# Setting Up an Amplify Full-Stack App
+This guide walks you through creating a full-stack application using Amplify. We'll cover the essential steps to get your project up and running.
```

Comment:
Can we separate it into following sections - 
1. Setting up CLI (which includes the access key part)
2. Setting up a new project
3. Setting up an existing project

And update the details accordingly?

Reply:
okay will update it